cloudify_nsx/library/nsx_esg_dlr.py

```diff
@@ -970,7 +970,30 @@ def esg_clear_interface(client_session, esg_id, ifindex):
 
 def update_common_edges(client_session, resource_id, kwargs, esg_restriction):
 
-    _, firewall = common.get_properties_and_validate('firewall', kwargs)
+    if esg_restriction:
```

Comment:
This appears to be assigning the same validation rules whether esg_restriction is true or not?